doc/circuitikzmanual.tex

```diff
@@ -3431,6 +3431,8 @@ \subsection{Block diagram components}
     \circuitdesc*{wilkinson}{wilkinson divider}{}( in/180/0.1, out2/45/0.1, out1/-45/0.1 )
     \circuitdesc*{splitter}{resistive splitter\footnotemark}{}( in/180/0.1, out2/45/0.1, out1/-45/0.1 )
     \footnotetext{added by \texttt{matthuszagh}}
+    \circuitdesc*{genericsplitter}{generic splitter\footnotemark}{}( in/180/0.1, out2/45/0.1, out1/-45/0.1 )
```

Comment:
Maybe (please test) `\circuitdesc*{genericsplitter}{generic splitter\footnotemark}{\SI{-3}{\dB}}` so users can see where to put the text?

Reply:
Done in 907a6cc. The example code renders a little strangely, it adds various spaces:
```
node[genericsplitter]{$\SI {-3}{\deci \bel }$}
```
I don't think this is a big issue though.